tooling/ssa_fuzzer/fuzzer/src/brillig.rs

```diff
@@ -69,16 +294,32 @@ libfuzzer_sys::fuzz_target!(|data: &[u8]| -> Corpus {
     let start = std::time::Instant::now();
     let fuzzer_output = fuzz_target(fuzzer_data, TARGET_RUNTIMES.to_vec(), options);
 
-    // If REDIS_URL is set and generated program is executed
-    if redis::ensure_redis_connection() {
-        // cargo-fuzz saves tests with name equal to sha1 of content
-        let mut hasher = Sha1::new();
-        hasher.update(data);
-        let sha1_hash = hasher.finalize();
-        let test_id = format!("{sha1_hash:x}");
-        match push_fuzzer_output_to_redis_queue("fuzzer_output", test_id, fuzzer_output) {
-            Ok(json_str) => log::debug!("{json_str}"),
-            Err(e) => log::error!("Failed to push to Redis queue: {e}"),
+    let transpiler: TranspileBrilligBytecodeToAbstractVMBytecode = Box::new(transpile);
```

Comment:
Please use init on fuzzer startup, so we don't get coverage information difference from the first input because of server initialization. https://github.com/rust-fuzz/libfuzzer/tree/main/example_init